src/main/rascal/refactor/Rename.rsc

```diff
@@ -44,6 +45,10 @@ import Set;
 
 alias RenameResult = tuple[list[DocumentEdit], set[Message]];
 
+// This leaves some room for more fine-grained steps should the user want to monitor that
+private int WORKSPACE_WORK = 10;
+private int FILE_WORK = 5;
```

Comment:
Should we expose those to the user via the `RenameConfig` as well? This allows the user to tweak the weights of the different steps, depending on implementation details.